src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -489,7 +496,7 @@
 
 helps['storage queue'] = """
     type: group
-    short-summary: Use queues to effectively scale applications according to traffic.
+    short-summary: Manage scaling and traffic routing for storage queues.
```

Comment:
None of the queue commands do any of this (manage scaling or traffic routing for storage queues).  It should really just be "Manage storage queues."

Reply:
I took that from the marketing copy; I agree it feels ambitious, but is there a way to do more than repeat the word in as its own definition?

Reply:
The original text was fine--it described what to use queues for, not what the commands do.  It's the new text that is misleading, because none of the queue commands involve scaling or routing. I just suggested the consistent alternative.